cronjob

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Perhaps this can be changed to `/usr/bin/env bash` instead, doing it like that will ensure that the script works on systems that doesn't have bash installed in /bin/bash